.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -43,7 +43,8 @@ class LoadTestsBuilder {
 
 
   static void loadTest(context, String title, Runner runner, SDK sdk, Map<String, ?> options,
-      String mainClass, List<String> jobSpecificSwitches = null, String requirementsTxtFile = null) {
+      String mainClass, List<String> jobSpecificSwitches = null, String requirementsTxtFile = null,
+      String pythonVersion = null) {
```

Comment:
I don't quite follow why we need these changes - could you explain them?

Reply:
I am using the TensorRT images provided by NVIDIA. This image has seen a good performance but they use python 3.8 in the container(this is the same image we use for TensorRT tests)

Since Python 3.7 is going to be deprecated soon, I thought we could run these tests on python 3.8.

Reply:
Sounds good - missed that 3.7 is the default. When we eventually deprecate 3.7 we should probably move all of these to our latest supported version (3.10 or 3.11) to reduce future churn.